XMTPiOSExample/XMTPiOSExample/Views/MessageComposerView.swift

```diff
@@ -30,12 +35,17 @@ struct MessageComposerView: View {
 	}
 
 	func send() {
+		if text.trimmingCharacters(in: .whitespacesAndNewlines) == "" {
```

Comment:
Is there any sort of "server side" validation to ensure we don't have empty messages or is that not really a concern?

Reply:
There is not currently and not sure if it's intentional or not. I've been [adding checks in the frontend](https://github.com/lensterxyz/lenster/blob/1d358d08ce150d20a281f2cd728d8d54754c4f06/apps/web/src/components/Messages/Composer.tsx#L24) to disable the send button until there's content (trimming whitespace like you're doing).

@mkobetic not urgent, but do you know if it's intentional that we allow blank messages?